modules/cbMap/validateMap.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Add VPL license and apply coreBOS [code formatting standards](http://corebos.org/documentation/doku.php?id=en:devel:developmentguidelines#code_formatting_and_validation_tools)